src/components/panel-tabs/index.js

```diff
@@ -41,7 +41,7 @@ const TABS = [
 const DEFAULT_TABS = [ 'layout', 'style', 'advanced' ]
 
 export const closeAllOpenPanels = clickedEl => {
-	[].forEach.call( document.querySelector( '.edit-post-sidebar, .edit-widgets-sidebar, .block-editor-block-inspector' )?.querySelectorAll( '.components-panel__body .components-panel__body-toggle' ) || [], el => {
+	[].forEach.call( document.querySelector( '.edit-post-sidebar, .edit-widgets-sidebar, .block-editor-block-inspector' )?.querySelectorAll( '.components-panel__body .components-panel__body-toggle .stk-panel' ) || [], el => {
```

Comment:
Is this correct? The component `Button` now has both classes: `components-panel__body-toggle stk-panel`, but the selector is a child selector? `.components-panel__body-toggle .stk-panel` (there's a space?)

Reply:
That's a mistake. I will fix that.